sphinx/tutorial/cylc/furthertopics/clock-triggered-tasks.rst

```diff
@@ -0,0 +1,179 @@
+.. include:: ../../../hyperlinks.rst
+   :start-line: 1
+
+Clock Triggered Tasks
+=====================
+
+This part of the Rose user guide walks you through using clock triggered tasks.
+
+This allows you to trigger tasks based on the actual time.
+
+
+Purpose
+-------
+
+.. TODO - Insert animation / diagram.
+
+The typical use of clock triggered tasks is to perform real-time data
```

Comment:
This is completely pedantic of me (once again) but there should really be a hyphen between 'clock' and 'triggered' when preceding 'tasks' and between 'wall' and 'clock' when preceding 'time'. This is probably too trivial to bother with but I might as well mention it while I am reviewing.

Reply:
I'll change them as and when I come accros them.